e2e/teams.spec.ts

```diff
@@ -52,9 +52,9 @@ let teamId: string | undefined = undefined
 // Run multiple describes in parallel, but run tests inside each describe in order
 // ...tests within describe expect afterAll to be run with the same users, i.e. on the same worker
 // https://playwright.dev/docs/api/class-test#test-describe-configure
-test.describe.configure({ mode: 'serial' })
+testAuth.describe.configure({ mode: 'serial' })
```

Comment:
Same in this file -- only changes are `test` -> `testAuth` and formatting changes